web/src/RealmApp.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import * as Realm from "realm-web";
+
+const RealmAppContext = React.createContext();
+
+export const useRealmApp = () => {
+  const app = React.useContext(RealmAppContext);
+  if (!app) {
+    throw new Error(
+      `You must call useRealmApp() inside of a <RealmAppProvider />`
```

Comment:
Nit: can we change the wording here to be similar to the other tutorial?

"useRealmApp() called outside of a RealmAppProvider?"

Reply:
I prefer the active voice here, particularly for a tutorial where the only user that should see this is a developer